app/Http/Controllers/RegulationController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\City;
+use App\County;
+use App\State;
+use App\Source;
+use App\Destination;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+
+Class RegulationController extends Controller{
+    public function allStates(){
+        $states = State::all();
+        return view("userSubmission", compact('states'));
+    }
+
+    public function getAllSources(){
+        return response()->json(Source::all());
+    }
+
+    public function getAllDestinations() {
+        return response()->json(Destination::all());
+    }
+
+    public function getCountiesInState(Request $request){
+        $counties = County::where("fk_state", $request->state_id)->get();
+        return response()->json($counties);
+    }
+
+    public function getCitiesInCounty(Request $request){
+        $cities = City::where("fk_county", $request->county_id)->get();
+        return response()->json($cities);
+    }
```

Comment:
See comment on `routes/api.php`. Since we're removing the extra API routes, we can delete these two functions as well.

Reply:
I could, but why? Just out of curiosity. I mostly wrote those to practice writing api's for laravel. I could get rid of them, but then that would be less lines of code written by me. I also like having all of my calls for the RegulationController being in one place. I'm down to change it, but not sure the reasoning. :)